tests/v2/validation/registries/registry_test.go

```diff
@@ -1,454 +0,0 @@
-package registries
-
-import (
-	"context"
-	"fmt"
```

Comment:
Why was this whole file removed?

Reply:
this was a mistake, I was waiting on Izaac's PR to merge to fix it.